app/models/coronavirus_restriction_search/devolved_nation_result.rb

```diff
@@ -0,0 +1,36 @@
+class CoronavirusRestrictionSearch::DevolvedNationResult
+  AREA_TYPES_FOR_NAME = %w[COI LBO LGD MTD UTA DIS].freeze
```

Comment:
`AREA_TYPES_FOR_NAME ` is too generic  as these are specifically lower tier area names. So can the old name, `LOWER_TIER_AREA_CODES` be reinstated as I think that better describes what they are.